diesel/src/connection/transaction_manager.rs

```diff
@@ -228,15 +234,11 @@ where
             .previous_error_relevant_for_rollback
             .take()
         {
-            // we can safely ignore the result of process_commit_error here
-            // as the only other error than the "rollback error" variants
-            // is failing to load the transaction state, but that's something
-            // we have already done above
-            let _ = process_commit_error(
+            let _: Error = process_commit_error(
```

Comment:
We likely want to keep/add some comment here why this error is ignored.

- [ ]